packages/sample-extension/README.md

```diff
@@ -0,0 +1,3 @@
+# Description
+
+This is a CommonJS webpack transpilation of the `node-fetch` package for use within `electron-renderer`.
```

Comment:
Please update to match sample-extension.

Reply:
Sounds good